src/pages/index.tsx

```diff
@@ -55,26 +55,27 @@ export function Index(props: RouteComponentProps) {
             listStyle: 'none',
             display: 'grid',
             gap: 3,
-            gridTemplateColumns: 'repeat(auto-fill, minmax(300px, 1fr))'
+            gridTemplateColumns: 'repeat(auto-fill, minmax(300px, 1fr))',
+            gridTemplateRows: '230px'
```

Comment:
Could probably get away with [`line-clamp`](https://caniuse.com/css-line-clamp) instead of doing this.

Reply:
I didn't know it could be used like that. Thank you 👏 .